cmd/populate-db/main.go

```diff
@@ -359,7 +359,7 @@ func defaultSettings(generatedAccountInfo generator.GeneratedAccountInfo, derive
 
 	settings.PreviewPrivacy = true
 	settings.Currency = "usd"
-	settings.ProfilePicturesVisibility = 1
+	settings.ProfilePicturesVisibility = 2
```

Comment:
it should be probably using some const that we defined somewhere (I know it's not your code, but mine :) ), would you mind pointing it at it?

thanks

Reply:
No problem, I changed it to use the constant.
However, to avoid shadowing I had to change the name `settings` used for the object here (there is a package under that name that is already imported). `defaultSettings` seems to be an appropriate name because that's exactly what we're creating